### PR TITLE
Second partv16update

### PR DESCRIPTION
**Pull request by kkeene44**
As part of updating the regression testing containers to v16, the file "Dockerfile-second_part" is updated to include the following modifications:

- Update maintainer from Dave Gill to Kelly Werner
- Updated to the latest built Docker container kkeene44/wrf-coop:version16
- Removed old commented-out code
- Updated the "git clone" command to include "--recurse-submodule," necessary for compiling and NoahMP
- Updated to now clone SCRIPTS/<various scripts> from the wrf-model/SCRIPTS repository, instead of davegill/SCRIPTS

These changes have all been tested, and along with PR #1, works well to build and run WRF.

### CODE REVIEW

**Comment by weiwangncar** [comment]
@kkeene44 Why do you need to add '--recurse-submodule' in the clone command? I thought the model would automatically obtain the files in submodule during the compile.

**Comment by kkeene44** [comment]
@weiwangncar 
The initial container I build only has all the libraries, compilers, and environment set up. I believe each time a reg test is run, it needs to clone the code again, but I'll look into it more to see if that's actually the case and get back to you before we approve/merge this.

**Comment by weiwangncar** [comment]
@kkeene44 I assume you've found the place the regtests get its container, and updated it to the new one?

**Comment by kkeene44** [comment]
> @kkeene44 I assume you've found the place the regtests get its container, and updated it to the new one?

@weiwangncar Yes, I have.

**Comment by kkeene44** [comment]
> @weiwangncar The initial container I build only has all the libraries, compilers, and environment set up. I believe each time a reg test is run, it needs to clone the code again, but I'll look into it more to see if that's actually the case and get back to you before we approve/merge this.

@weiwangncar It looks like this "git clone" command is only for testing the newly-built container each time - prior to giving it to Scala. We would build a brand new container from the image that only contains the set-up environment (minus WRF installation), so the "--recurse-submodule" option would need to be included since this would be the first time the code is checked out in this test container.

**Comment by weiwangncar** [comment]
@kkeene44 OK, that sounds good to me.

**Comment by kkeene44** [comment]
@weiwangncar Can you approve it? I set these up so that they only need a single approval for now.

**Review by weiwangncar — APPROVED** [review]

**Pull request merged**
